src/Parsers/ASTSystemQuery.cpp

```diff
@@ -224,6 +224,11 @@ void ASTSystemQuery::formatImpl(const FormatSettings & settings, FormatState &,
             }
         }
     }
+    else if (type == Type::DROP_SCHEMA_CACHE)
+    {
+        if (!schema_cache_storage.empty())
+            settings.ostr << (settings.hilite ? hilite_none : "") << " FOR " << backQuoteIfNeed(schema_cache_storage);
```

Comment:
FOR isn't highlighted as a keyword.

Reply:
Right, but I did not want to touch anything except for this query. But OK, let's fix everything else, since right now there are multiple ways of printing identifiers/keywords.